com.makemkv.MakeMKV.json

```diff
@@ -0,0 +1,121 @@
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  /* NOTE: java is optional to help find blu-ray main titles */
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk10"
+  ],
+  "finish-args": [
+    "--filesystem=home",
+    /* X11 access */
+    "--socket=x11",
+    "--share=ipc",
+    /* Network access */
+    "--share=network",
+    /* For optical drive access */
+    "--device=all",
+    "--env=JAVA_HOME=/app/jre"
+  ],
+  "command": "makemkv",
+  "build-options" : {
+    "append-path": "/usr/lib/sdk/openjdk10/bin"
+  },
+  "rename-desktop-file": "makemkv.desktop",
+  "rename-icon": "makemkv",
+  "cleanup": [
+    "/include",
+    "/lib/debug",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "openjdk",
+        "buildsystem": "simple",
+        "build-commands": [
+          "/usr/lib/sdk/openjdk10/install.sh"
+        ]
+    },
+    {
+      "name": "x264",
+      "config-opts": [
+        "--disable-cli",
+        "--enable-shared"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.videolan.org/x264/snapshots/x264-snapshot-20171223-2245-stable.tar.bz2",
+          "sha256": "ee4ba2c3d1caf786ad66d3fa86bdc368a8f827318788a62a9c8444b2c35816bf"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "config-opts": [
+        "--enable-gpl",
+        "--disable-static",
+        "--enable-shared",
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-ffprobe",
+        "--enable-libopus",
+        "--enable-libvpx",
+        "--enable-libx264"
+      ],
+      "post-install": [
+        "chrpath -d /app/bin/ffmpeg"
+      ],
+      "cleanup": [
+        "/share/ffmpeg/examples"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ffmpeg.org/releases/ffmpeg-4.0.2.tar.xz",
+          "sha256": "a95c0cc9eb990e94031d2183f2e6e444cc61c99f6f182d1575c433d62afb2f97"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-oss",
+      "buildsystem": "simple",
+      "build-commands": [
+        "export CFLAGS='-D__GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS'",
+        "./configure --enable-debug --enable-allcodecs --prefix=/app",
+        "make -j $FLATPAK_BUILDER_N_JOBS",
+        "make install DESTDIR=/"
+      ],
+      "sources" : [
+        {
+          "type": "archive",
+          "url": "http://www.makemkv.com/download/makemkv-oss-1.14.1.tar.gz",
+          "sha256": "fc8e15ef1148988b241110fc7cf2c2d916abf01bb85c3450b4e2ee10aa054b7b"
+        },
+        {
+          "type": "file",
+          "path": "com.makemkv.MakeMKV.appdata.xml"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-bin",
+      "buildsystem": "simple",
+      "build-commands": [
+        /* TODO: put eula in appstream data once it supports it */
+        "mkdir -p tmp",
+        "echo accepted > tmp/eula_accepted",
```

Comment:
I'm not sure. Not that this is reason alone, but it's pretty widely distributed:

* https://negativo17.org/repos/multimedia/fedora-29/x86_64/makemkv-1.12.3-1.fc29.x86_64.rpm
* https://packages.gentoo.org/packages/media-video/makemkv
* https://aur.archlinux.org/packages/makemkv/
* https://launchpad.net/~heyarje/+archive/ubuntu/makemkv-beta

....

After searching for the forums, I found this post:

https://www.makemkv.com/forum/viewtopic.php?f=3&t=3437&p=12765#p12765

It looks like as long as the package description references to eula, the author is ok with redistribution.

That being said, it would be really nice if appstream implemented eula support (https://github.com/ximion/appstream/issues/50)